app/resources/v1/sample_resource.rb

```diff
@@ -43,6 +43,19 @@ class SampleResource < JSONAPI::Resource
     attributes :sanger_sample_id, :supplier_name, :taxon_id, :donor_id, :country_of_origin,
                :accession_number, :date_of_sample_collection
 
+    filter :name
+    attribute :pacbio_requests
```

Comment:
I think we should be able to do pacbio_requests as a relationship through request as opposed to an attribute. Do we need ont_requests too?

Reply:
Should we do this the other way round? Should we use requests and pull in the sample information? That way we could have a different report for Pacbio and ONT (if required) to reduce complexity especially since they are so different?